server/src/main/java/keywhiz/KeywhizService.java

```diff
@@ -119,10 +120,7 @@ public void setInjector(Injector injector) {
   }
 
   private void doRun(KeywhizConfig config, Environment environment) {
-    if (injector == null) {
-      logger.debug("No existing guice injector; creating new one");
-      injector = Guice.createInjector(new ServiceModule(config, environment));
-    }
+    ensureInjectorCreated(config, environment);
 
```

Comment:
I wanted to be able to test this behavior in isolation without all the other side effects of running the service, so I moved it into a helper method.